__tests__/integration.test.js

```diff
@@ -85,4 +105,9 @@ describe("Error handing", () => {
 				});
 		});
 	});
+	describe("3. GET /api/users", () => {
+		test("should respond with status 404", () => {
```

Comment:
Just be a bit more descriptive with your test's please, should respond with a status 404 if what?

Reply:
sure thing, will update before merge, thanks !